code/mod_table/mod_table.cpp

```diff
@@ -380,16 +382,33 @@ void parse_mod_table(const char *filename)
 
 		if (optional_string("$Generic Pain Flash Factor:")) {
 			stuff_float(&Generic_pain_flash_factor);
-			if (Generic_pain_flash_factor != 1.0f)
+			if (!fl_equal(Generic_pain_flash_factor, 1.0f))
 				mprintf(("Game Settings Table: Setting generic pain flash factor to %.2f\n", Generic_pain_flash_factor));
 		}
 
 		if (optional_string("$Shield Pain Flash Factor:")) {
 			stuff_float(&Shield_pain_flash_factor);
-			if (Shield_pain_flash_factor != 0.0f)
+			if (!fl_near_zero(Shield_pain_flash_factor, 0.01f))
 				 mprintf(("Game Settings Table: Setting shield pain flash factor to %.2f\n", Shield_pain_flash_factor));
 		}
 
+		if (optional_string("$EMP Pain Flash Factor:")) {
+			stuff_float(&Emp_pain_flash_factor);
+			if (!fl_near_zero(Emp_pain_flash_factor, 0.01f))
+				mprintf(("Game Settings Table: Setting EMP pain flash factor to %.2f\n", Emp_pain_flash_factor));
+		}
+
+		if (optional_string("$EMP Pain Flash Color:")) {
+			int rgb[3];
+			stuff_int_list(rgb, 3);
+			if ((rgb[0] >= 0 && rgb[0] <= 255) && (rgb[1] >= 0 && rgb[1] <= 255) && (rgb[2] >= 0 && rgb[2] <= 255)) {
+				Emp_pain_flash_color = std::make_tuple(static_cast<float>(rgb[0]/255), static_cast<float>(rgb[1]/255), static_cast<float>(rgb[2]/255));
```

Comment:
The cast to float should be done before the division, otherwise you'll be dividing an int by an int which in this case will give you only 0 or 1.